spk/sudo/Makefile

```diff
@@ -0,0 +1,25 @@
+SPK_NAME = sudo
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/sudo.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = frv
+DESCRIPTION = "Sudo allows a system administrator to delegate authority to give certain users (or groups of users) the ability to run some (or all) commands as root or another user while providing an audit trail of the commands and their arguments."
+RELOAD_UI = no
```

Comment:
Honestly, I've no clue on what this is doing and  'have not spent much time to find it. If anyone can give me an explanation on this field ... 
By the way, don't specify it  seems hurtless.